### DISCUSSION TITLE
[FeatureRequest] TextPanel: Adding emoji support

### DISCUSSION THREAD

**Discussion by yarix** [grafana/grafana#38291] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
add support for emoji on text panel when using Markdown text mode.
eg. `:smile:` should render to :smile: 
**Why is this needed**:
allow to create better text messages

**Comment by jan25** [comment]
Seems like feature is missing in 'marked' dependency library used to render markdown. Related issue: https://github.com/markedjs/marked/issues/2061

**Comment by yarix** [comment]
another missing functionality in the markdown is embedded images. for example:

```
Here's an image:

![][image_ref_a32ff4ads]

More text here...
...

[image_ref_a32ff4ads]: data:image/png;base64,iVBORw0KGgoAAAANSUhEke02C1MyA29UWKgPA...RS12D==

```

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
